### PR TITLE
techpowerup.com

### PR DESCRIPTION
**Pull request by SKEIDs**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/en/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 https://www.techpowerup.com/

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1946,6 +1946,7 @@ indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-pa
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
 linuxadictos.com#?##content > aside[id^="text-"]:has(> div.textwidget > p.gt-block > ins.adsbygoogle)
+techpowerup.com#?#.v-port > div:has(>div > img[src*="/reviyuu"])
```
Comment (Yuki2718) [inline BaseFilter/sections/css_extended.txt:1949]: Better `#?#.v-port > div:has(> a[rel="nofollow"][href^="/reviyuu/"])`. Make the depth as few as possible, i.e. (> * > *) => (> *). Also using src for :has is bit risky as the src may be removed from source code (https://github.com/AdguardTeam/AdguardFilters/issues/95975#issuecomment-961235146)

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/css_extended.txt (hunk 1 above)
Comment (SKEIDs) [reply BaseFilter/sections/css_extended.txt:1949]: I see. I will fix rule.

**Review by Yuki2718 — APPROVED** [review]

File: BaseFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -1946,6 +1946,7 @@ indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-pa
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
 linuxadictos.com#?##content > aside[id^="text-"]:has(> div.textwidget > p.gt-block > ins.adsbygoogle)
+techpowerup.com#?#.v-port > div:has(> a[rel="nofollow"][href^="/reviyuu/"])
```
Comment (zloyden) [inline BaseFilter/sections/css_extended.txt:1949]: Where is this rule needed?

File: BaseFilter/sections/specific.txt [hunk 3]
```diff
@@ -2019,6 +2019,7 @@ oploverz.asia##div[id^="oplov-"]
 oploverz.asia###sidebar > div.widget_text
 chillx.top###theotherads
 ||www.techpowerup.com/reviyuu/
+techpowerup.com##.s--item[br-cls]
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:2022]: Looks like they start circumvent the rules. It also should be reported in https://github.com/AdguardTeam/AdGuardExtra.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/specific.txt (hunk 3 above)
Comment (SKEIDs) [reply BaseFilter/sections/specific.txt:2022]: I see, thank you for post issue for AdGuardExtra.

File: BaseFilter/sections/css_extended.txt (hunk 2 above)
Comment (SKEIDs) [reply BaseFilter/sections/css_extended.txt:1949]: I think the website design have changed.
so, the rule has to removed.

File: BaseFilter/sections/specific.txt (hunk 3 above)
Comment (SKEIDs) [reply BaseFilter/sections/specific.txt:2022]: Should I close this pull request if adguard extra to handle?

Comment (zloyden) [reply BaseFilter/sections/specific.txt:2022]: Yep, I'll close this due to https://github.com/AdguardTeam/AdGuardExtra/issues/401.

Comment (SKEIDs) [reply BaseFilter/sections/specific.txt:2022]: ok,thank you.

**Pull request closed without merging**